dluhc-component-library/src/components/maps/mapContainer.tsx

```diff
@@ -21,7 +21,9 @@ const MapContainer = ({
   style,
   children,
 }: MapContainerProps) => {
-  const [map] = useState(new OLMap());
+  const [map] = useState(
+    new OLMap({ view: new View({ center: [0, 0], zoom: 0 }) }),
```

Comment:
This adds a default view so that child components don't error if they try and get data about the map dimensions. We could also lift the view state up into the container instead with the `BaseMap` only responsible for adding the tile layer- similar to React Leaflet: https://react-leaflet.js.org/